<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,1">
   <item row="1" column="0">
    <widget class="QGroupBox" name="plot_groupBox">
     <property name="title">
      <string>Plot</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="settings_groupBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="interrupt_pushButton">
        <property name="text">
         <string>Interrupt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="start_pushButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="QGroupBox" name="Live_groupBox">
        <property name="minimumSize">
         <size>
          <width>349</width>
          <height>65</height>
         </size>
        </property>
        <property name="title">
         <string>Live</string>
        </property>
        <widget class="QCheckBox" name="autoLevels_checkBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>76</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>autoLevels</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="min_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="max_doubleSpinBox">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>20</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_max">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>20</y>
           <width>21</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>min</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_min">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>20</y>
           <width>28</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>max</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="autoRange_checkBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>75</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>autoRange</string>
         </property>
        </widget>
        <zorder>label_max</zorder>
        <zorder>autoLevels_checkBox</zorder>
        <zorder>min_doubleSpinBox</zorder>
        <zorder>max_doubleSpinBox</zorder>
        <zorder>label_min</zorder>
        <zorder>autoRange_checkBox</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
